WordPress/Classes/ViewRelated/NUX/ContentViews/Editor/UnifiedPrologueEditorContentView.swift

```diff
@@ -0,0 +1,120 @@
+import SwiftUI
+
+/// Prologue editor page contents
+struct UnifiedPrologueEditorContentView: View {
+
+    var body: some View {
+        GeometryReader { content in
+            VStack {
+                Spacer(minLength: content.size.height * 0.18)
```

Comment:
Normally I would comment on magic numbers, but I think in this situation it's fine as it's just specific ratios for placement in a visual layout and I'm not sure having names for them would be any more descriptive.